spec/clean-spec.js

```diff
@@ -1,98 +1,74 @@
-    app.get('/test-module', (request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'install-test-module.json')));
-    app.get('/tarball/test-module-1.2.0.tgz', (request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'test-module-1.2.0.tgz')));
-
+    app.get(`/node/${nodeVersion}/win-x86/node.lib`, (_request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'node-dist', 'node.lib')));
+    app.get(`/node/${nodeVersion}/win-x64/node.lib`, (_request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'node-dist', 'node_x64.lib')));
+    app.get(`/node/${nodeVersion}/SHASUMS256.txt`, (_request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'node-dist', 'SHASUMS256.txt')));
+    app.get('/test-module', (_request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'install-test-module.json')));
+    app.get('/tarball/test-module-1.2.0.tgz', (_request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'test-module-1.2.0.tgz')));
     server = http.createServer(app);
-
     let live = false;
-    server.listen(3000, '127.0.0.1', function() {
-      console.log("Server started");
-      const atomHome = temp.mkdirSync('apm-home-dir-');
-      process.env.ATOM_HOME = atomHome;
-      process.env.ATOM_ELECTRON_URL = "http://localhost:3000/node";
+    server.listen(3000, '127.0.0.1', () => {
+      console.log('Server started');
+      process.env.ATOM_HOME = temp.mkdirSync('apm-home-dir-');
+      process.env.ATOM_ELECTRON_URL = 'http://localhost:3000/node';
       process.env.ATOM_ELECTRON_VERSION = nodeVersion;
       process.env.npm_config_registry = 'http://localhost:3000/';
-
       moduleDirectory = path.join(temp.mkdirSync('apm-test-module-'), 'test-module-with-dependencies');
       wrench.copyDirSyncRecursive(path.join(__dirname, 'fixtures', 'test-module-with-dependencies'), moduleDirectory);
       process.chdir(moduleDirectory);
-      return live = true;
+      live = true;
     });
-    return waitsFor(() => live);
-  });
+    waitsFor(() => live);
+  })
 
-  afterEach(function() {
-    let done = false;
-    server.close(() => done = true);
-    return waitsFor(() => done);
+  afterEach(() => {
+    let done = false
+    server.close(() => {
+      done = true;
+    });
+    waitsFor(() => done);
   });
 
-  it('uninstalls any packages not referenced in the package.json', function() {
+  it('uninstalls any packages not referenced in the package.json', () => {
     const removedPath = path.join(moduleDirectory, 'node_modules', 'will-be-removed');
     fs.makeTreeSync(removedPath);
-    fs.writeFileSync(
-      path.join(removedPath, 'package.json'),
-      '{"name": "will-be-removed", "version": "1.0.0", "dependencies": {}}',
-      'utf8'
-    );
-
+    fs.writeFileSync(path.join(removedPath, 'package.json'), '{"name": "will-be-removed", "version": "1.0.0", "dependencies": {}}', 'utf8');
```

Comment:
Maybe 5 lines is a lot for something as simple as writing out some JSON to a file on disk. But I do think the indented and spaced out version was easier to read before being condensed onto one line.

I would not block this PR from merging for this, but if it were me I would leave the indentation for readability (easier to skim quickly IMO).

If you do simply undo this whitespace-only change, I would re-approve as soon as I see the notification (which may not be right away) but other reviewers can presume I am 👍 approving of said change.

Reply:
Okay, fair enough I think, I have undone this change.